oct_converter/readers/fda.py

```diff
@@ -224,7 +246,8 @@ def read_oct_volume(self):
         """
 
         if b"@IMG_JPEG" not in self.chunk_dict:
-            raise ValueError("Could not find OCT header @IMG_JPEG in chunk list")
+            print("@IMG_JPEG IS NOT IN CHUNKS SKIPPING...")
```

Comment:
Being pedantic but could we change to `print("Could not find OCT header @IMG_JPEG in chunk list, skipping.")` I think the capitals make it look like we're shouting :P

Reply:
Oh okay I did not think like that :D